packages/bento-design-system/src/Charts/Tooltip/Tooltip.tsx

```diff
@@ -3,15 +3,21 @@ import { Box } from "../../Box/Box";
 import { Column, Columns } from "../../Layout/Columns";
 import { Stack } from "../../Layout/Stack";
 import { Body } from "../../Typography/Body/Body";
+import { NameType, ValueFormatter, ValueType } from "../utils";
 
-type ValueType = number | string;
-type NameType = number | string;
+type TooltipProps<TValue extends ValueType, TName extends NameType> = RechartsTooltipProps<
+  TValue,
+  TName
+> & {
+  valueFormatter?: ValueFormatter;
+};
```

Comment:
I think we can use the standard `formatter` value from `Tooltip`. If we do that we can avoid this intermediate `TooltipComponent` wrapping and keep using `tooltipComponent` as before, and passing `yAxisValueFormatter` to `Tooltip` directly.

I have a working prototype (based on this branch) here: https://github.com/buildo/bento-design-system/compare/472-add_support_for_units-2?expand=1

Let me know what you think